### PR TITLE
Adds LabReport, LabValue and related models.

### PR DESCRIPTION
**Pull request by jrwils**
https://canvasmedical.atlassian.net/browse/KOALA-1843

### CODE REVIEW

File: canvas_sdk/v1/data/lab.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+    dbid = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField()
+    modified = models.DateTimeField()
+    value = models.ForeignKey(
+        LabValue, on_delete=models.DO_NOTHING, related_name="codings", null=True
+    )
+    code = models.CharField(max_length=128, blank=True)
+    name = models.CharField(max_length=256, blank=True)
+    # there is no system in the database; these are all assumed to be LOINC codes
```
Comment (aduane) [inline canvas_sdk/v1/data/lab.py:105]: If we add
```sql
SELECT "http://loinc.org" AS system
```
to the view we don't have to treat this any differently w/r/t code lookups.

File: canvas_sdk/v1/data/lab.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_labreport_001"
+
+    objects = CommittableModelManager()
+
+    id = models.UUIDField()
+    dbid = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField()
+    modified = models.DateTimeField()
+    review_mode = models.CharField(max_length=2)
```
[the remark this answers is not in the document]
Comment (jrwils) [reply canvas_sdk/v1/data/lab.py:20]: I'm used to max_length being a required argument. Maybe that has changed in more recent versions. Looks like it doesn't complain, so I'll take the unnecessary arguments out.

File: canvas_sdk/v1/data/lab.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+class LabReview(models.Model):
+    class Meta:
+        managed = False
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_labreview_001"
+
+    objects = CommittableModelManager()
+
+    # id = models.UUIDField() # TODO - add externally_exposable_id in home-app
```
[the remark this answers is not in the document]
Comment (jrwils) [reply canvas_sdk/v1/data/lab.py]: Added.

File: canvas_sdk/v1/data/lab.py (hunk 1 above)
Comment (jrwils) [reply canvas_sdk/v1/data/lab.py]: Done.

File: canvas_sdk/v1/data/lab.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jrwils) [reply canvas_sdk/v1/data/lab.py]: Done.

**Review by aduane — APPROVED** [review]

**Pull request merged**